data/kunal00000.json

```diff
@@ -46,5 +37,30 @@
       "url": "https://www.instagram.com/kunaahl",
       "icon": "FaInstagram"
     }
+  ],
+  "milestones": [
+    {
+      "title": "Postman Student Expert",
+      "date": "May 2023",
+      "icon": "SiPostman",
+      "color": "orange",
+      "description": "Became a Postman Student Expert after completing the API Fundamentals",
```

Comment:
```suggestion
      "description": "Became a Postman Student Expert after completing the API Fundamentals"
```

Reply:
Hi, 
Thanks for pointing out Pradumna, commited the changes.
Btw how did you write this suggestion, can you share text of this 🙂.